snuba/web/query.py

```diff
@@ -185,7 +186,13 @@ def _run_query_pipeline(
         metrics.increment("sample_without_turbo", tags={"referrer": request.referrer})
 
     if request.settings.get_dry_run():
-        query_runner = _dry_run_query_runner
+        storage_names = []
```

Comment:
Why do we need to collect storage names here? Isn't that the job of the storage selector later in the query pipeline?

Reply:
Same reason as above